packages/web-app-files/src/store/actions.ts

```diff
@@ -205,6 +204,9 @@ export default {
             { root: true }
           )
         })
+        .finally(() => {
+          setProgress({ total: files.length, current: removedFiles.length })
```

Comment:
Hm - looking at this again: a file is pushed only to the removedFiles after success, right? So just moving this to finally should have no effect I guess.

Reply:
Or simply do it like in the other places via `for (const [i, file] of files.entries())` and use `i`?